changes

Fisheiyy opened the pull request:
minor changes to follow in the path of changes that directly impact mcreator

Fisheiyy committed:
updates

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/MCreator/Generator-Forge-1.14.4?pullRequest=3) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/MCreator/Generator-Forge-1.14.4?pullRequest=3) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: Fisheiyy<br/>:x: KlemenDEV<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/MCreator/Generator-Forge-1.14.4?pullRequest=3) it.</sub>

KlemenDEV commented on forge-1.14.4/workspacebase/build.gradle:
```diff
@@ -15,10 +14,10 @@ version = '1.0'
 
 minecraft {
-    mappings channel: 'snapshot', version: '20190719-1.14.3'
+    mappings channel: 'snapshot', version: '20200119-1.14.4'
```
Mappings should not be changed like this as names and mappings system in the generator may break. Keep existing mappings.

Fisheiyy replied:
it honestly might be best though to go through and update to newer mappings since this is written technically as a 1.14.3 generator or atleast the mappings are, will look into this though but for now i will revert

Fisheiyy committed:
revert changes

java seems to not recognize itself will look into considering 1.16.5 generator works just fine

Fisheiyy committed:
revert java version check

Fisheiyy committed:
update gradle

KlemenDEV commented on forge-1.14.4/workspacebase/gradle/wrapper/gradle-wrapper.properties:
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.1-all.zip
```
If you update the wrapper version, update the wrapper JAR too.

Fisheiyy committed:
Update gradle-wrapper.jar

KlemenDEV reviewed (changes requested):
Checking again this version of forge gradle does not work with the gradle wrapper you provided, please revert gradle wrapper update.

Fisheiyy committed:
revert gradle

Fisheiyy committed:
Update mods.toml.ftl

Fisheiyy commented:
i could update forge gradle along with gradle but im not sure any newer version of forge gradle is compatible with this version of minecraft

Fisheiyy committed:
revert mods.toml.ftl, update generator.yaml

KlemenDEV committed:
Update generator.yaml

KlemenDEV committed:
Update build.gradle

KlemenDEV committed:
Update build.gradle

KlemenDEV committed:
Update plugin.json

Fisheiyy commented:
oh uhh hey make sure to make a 2021.1 branch and delete 2020.6 also why add a subversion for 1.12.2 generator but not this one like my original pull wanted to do?

KlemenDEV commented:
There is no need for subversion here, also not in 1.12.2 but it slipped in with my PR.
